src/recensio/plone/browser/review.py

```diff
@@ -348,7 +349,11 @@ def get_metadata_context_object(self):
                         {
                             name: [
                                 f"{au.firstname} {au.lastname}"
-                                for au in [au.to_object for au in context.reviewAuthors]
+                                for au in [
+                                    au.to_object
+                                    for au in context.reviewAuthors
+                                    if au and au.to_object
```

Comment:
You could have sued "filter None" here as well to save a double call to `to_object`.
Given that we already see performance issues, I am really in favor of that.
But I not consider this as a blocker.